data/test/hf_transformers_models.txt

```diff
@@ -4,31 +4,51 @@ https://s3.amazonaws.com/models.huggingface.co/bert/rasa/LaBSE/special_tokens_ma
 99497d78492c90ab7d824d695b9a8d043369fbc2bf1112dcc7cdef9a6c4fa691.275045728fbf41c11d3dae08b8742c054377e18d92cc7b72b6351152a99b64e4
 https://s3.amazonaws.com/models.huggingface.co/bert/rasa/LaBSE/tokenizer_config.json
 527f618330e845c9d31826e7d9ce983aa816fafcf4f29f8c52f8ae1fdd097219.1c61d5d3dc67d88e0c74c64cda9b17bc30bdbd1c373cceeb740b9953729709aa
+https://s3.amazonaws.com/models.huggingface.co/bert/rasa/LaBSE/config.json
+90984a8da5021905af8679644b61bc5428ef16e9a307469152c163ec873db240.f1ba7080a92fc164a144311742f36dfb6a724bc9da532264b30d87040e15cc9d
```

Comment:
I was unsure about this, I felt like it would be misleading to not have them in there since people might run the `prepare-transformers` target **locally** and expect it to prepare for all of the tests. I've adjusted the target so it doesn't download those files on the CI